### PR TITLE
Cleanup prebuild script

### PR DESCRIPTION
**Pull request by lgeiger**
Main changes:
- Replace `fs.stat` with `fs.existsSync`
- Use `noop-log` in tests

### CODE REVIEW

File: prebuild.js [hunk 1]
```diff
@@ -17,45 +17,39 @@ function prebuild (opts, target, runtime, callback) {
...
-      },
-      function (filename, cb) {
-        buildLog('Packing ' + filename + ' into ' + tarPath)
-        pack(filename, tarPath, function (err) {
-          if (err) return cb(err)
-          cb(null)
-        })
-      }
-    ]
+  if (fs.existsSync(tarPath) && !opts.force) {
```
Comment (mafintosh) [inline prebuild.js]: whats the benefit to using existsSync over stat here? function is async anyway

Comment (lgeiger) [inline prebuild.js]: Personally I think it only makes the code more readable.

There are also discussions about why `existsSync` may be better/faster: https://github.com/nodejs/node/pull/8364

Comment (mafintosh) [inline prebuild.js]: I don't think we should start using sync calls in async methods. It's a bad pattern.

**Comment by mafintosh** [comment]
+1 except for the sync call

File: prebuild.js (hunk 1 above)
Comment (lgeiger) [inline prebuild.js]: 👍 Removed the last commit.

**Comment by mafintosh** [comment]
👍 Thanks! Do you wanna help maintain this in general?

**Comment by mafintosh** [comment]
5.0.1

**Comment by lgeiger** [comment]
>  Do you wanna help maintain this in general?

Sure, I'm happy to help 🎉

File: prebuild.js [hunk 2]
```diff
@@ -48,13 +49,7 @@ function prebuild (opts, target, runtime, callback) {
...
     }
 
-    // TODO if we can move out buildLog() to the caller, we can simply do
-    // async.waterfall(tasks, callback)
-    async.waterfall(tasks, function (err) {
-      if (err) return callback(err)
-      buildLog('Prebuild written to ' + tarPath)
-      callback(null, tarPath)
-    })
+    async.waterfall(tasks, callback)
```
Comment (ralphtheninja) [inline prebuild.js]: Nice!

**Pull request merged**